Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Jamulus/Startpage](https://hosted.weblate.org/projects/jamulus/1-index/).


It also includes the following components:

* [Jamulus/Navigation](https://hosted.weblate.org/projects/jamulus/navigation/)

* [Jamulus/Installation-for-Macintosh](https://hosted.weblate.org/projects/jamulus/installation-for-macintosh/)

* [Jamulus/Copyright notices](https://hosted.weblate.org/projects/jamulus/copyright-notices/)

* [Jamulus/Include-Backing-Up](https://hosted.weblate.org/projects/jamulus/include-backing-up/)

* [Jamulus/Server-Bandwidth](https://hosted.weblate.org/projects/jamulus/server-bandwidth/)

* [Jamulus/Software-Manual](https://hosted.weblate.org/projects/jamulus/software-manual/)

* [Jamulus/FAQ](https://hosted.weblate.org/projects/jamulus/faq/)

* [Jamulus/Server-Troubleshooting](https://hosted.weblate.org/projects/jamulus/server-troubleshooting/)

* [Jamulus/Include-Shared-Commands](https://hosted.weblate.org/projects/jamulus/include-shared-commands/)

* [Jamulus/Installation-for-Android](https://hosted.weblate.org/projects/jamulus/installation-for-android/)

* [Jamulus/QOS-Windows](https://hosted.weblate.org/projects/jamulus/qos-windows/)

* [Jamulus/Command-Line-Options](https://hosted.weblate.org/projects/jamulus/command-line-options/)

* [Jamulus/Installation-for-iOS](https://hosted.weblate.org/projects/jamulus/installation-for-ios/)

* [Jamulus/Tips-Tricks-More](https://hosted.weblate.org/projects/jamulus/tips-tricks-more/)

* [Jamulus/Getting-Started](https://hosted.weblate.org/projects/jamulus/getting-started/)

* [Jamulus/Client-Troubleshooting](https://hosted.weblate.org/projects/jamulus/client-troubleshooting/)

* [Jamulus/Include-Client-Commands](https://hosted.weblate.org/projects/jamulus/include-client-commands/)

* [Jamulus/General strings](https://hosted.weblate.org/projects/jamulus/general/)

* [Jamulus/Contribution](https://hosted.weblate.org/projects/jamulus/contribution/)

* [Jamulus/Unregistered-Servers](https://hosted.weblate.org/projects/jamulus/unregistered-servers/)

* [Jamulus/Installation-for-Linux](https://hosted.weblate.org/projects/jamulus/installation-for-linux/)

* [Jamulus/Installation-for-Windows](https://hosted.weblate.org/projects/jamulus/installation-for-windows/)

* [Jamulus/Running-a-Server](https://hosted.weblate.org/projects/jamulus/running-a-server/)

* [Jamulus/Directories](https://hosted.weblate.org/projects/jamulus/directories/)

* [Jamulus/Privacy-Statement](https://hosted.weblate.org/projects/jamulus/privacy-statement/)



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/jamulus/1-index/horizontal-auto.svg)

ann0see commented on _translator-files/po/ko_KR/Unregistered-Servers.po:
```diff
@@ -42,7 +42,7 @@ msgstr ""
 #: ../wiki/en/Unregistered-Servers.md:9
 msgid "{% include breadcrumb.html root=\"More\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"더보기\" branch1=\"서버 관리\" branch1-url=\"서버 실행\" %}"
```
This is wrong! The URL must not be translated. Otherwise we'd have a broken link!

pljones replied:
Looks like you fixed it?  Can it go into 3.10.0?

ann0see replied:
Yes. We'll still need to check it.

ann0see reviewed (approved):
Ok. After looking through the diff - it seems to be ok now.

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/
Translation: Jamulus/Tips-Tricks-More

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Translation: Jamulus/Unregistered-Servers
Translate-URL: https://hosted.weblate.org/projects/jamulus/unregistered-servers/

weblate committed:
Update translation files

Updated by "Squash Git commits" hook in Weblate.

Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://hosted.weblate.org/projects/jamulus/tips-tricks-more/
Translation: Jamulus/Tips-Tricks-More

ann0see commented on _translator-files/po/es/Running-a-Server.po:
```diff
@@ -7,15 +7,15 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: \n"
-"PO-Revision-Date: 2023-08-18 06:52+0000\n"
```
@ignotus666 please have a look at this.

ann0see commented on _translator-files/po/ko_KR/FAQ.po:
```diff
@@ -92,142 +90,139 @@ msgstr "**Jamulus를 사용할 때 발생하는 일반적인 문제와 해결 
 #: ../wiki/en/FAQ.md:38
 msgid "Whether or not you will be able to play in time with other musicians depends mainly on how much latency (delay) you have in your sound signal and whether you are all following [The Golden Rule](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together). An overall delay much bigger than 50 ms usually makes it too hard to play in time unless you train yourself to do s ...
-msgstr ""
+msgstr "다른 뮤지션들과 함께 제 시간에 연주할 수 있는지 여부는 주로 사운드 신호의 레이턴시(딜레이) 얼마나 되는지, 그리고 모두가 [황금률](/wiki/Client-Troubleshooting#you-all-sound-ok-but-its-difficult-to-keep-together)을 따르고 있는지 여부에 따라 달라집니다. 50ms보다 훨씬 큰 전체 딜레이는 일반적으로 훈련하지 않는 한 시간에 연주하는 것을 너무 어렵게 만듭니다. 일부 뮤지션들은 비트보다 앞서 연주함으로써 최대 70ms의 딜레이에 맞춰 연주할 수 있다고 보고합니다. 하지만 그 이상이면 타이밍이 음악에서 중요한 요소가 아니거나 단순히 듣기만 원하는 경우가 아니라면 너무 어려워집니다."
```
Just as quick note - the links will not be broken, but the anchor just fail. I'm unsure how the actual ID would look like for non ASCII headlines.

ann0see commented on _translator-files/po/ko_KR/Running-a-Server.po:
```diff
@@ -316,24 +292,18 @@ msgstr ""
 "<figure>\n"
-"\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Jamulus 등록 서버 내 클라이언트 간 연결 다이어그램\">\n"
-"<figcaption>등록된 서버의 작동 방법</figcaption>\n"
+"\t<img src=\"{% include img/en-screenshots/diagram-reg-server.inc %}\" loading=\"lazy\" alt=\"Jamulus 등록된 서버 내의 클라이언트 간 연결 다이어그램\">\n"
```
If we have ko_KR screenshots this would be nice. I'd always prefer not to use the screenshots as includes.

ann0see commented on _translator-files/po/ko_KR/Tips-Tricks-More.po:
```diff
@@ -323,6 +343,19 @@ msgid ""
 "?>\n"
 "<?php loadserverstat ( \"stat1.dat\" ); ?>\n"
 msgstr ""
+"<?php\n"
```
The script should really go into the KB...

ann0see reviewed (commented):
Looks ok. Please review and I'll soon merge it.
